runtimes/peregrine/src/lib.rs

```diff
@@ -685,6 +685,105 @@ impl pallet_utility::Config for Runtime {
 
 impl pallet_randomness_collective_flip::Config for Runtime {}
 
+// Start of proxy pallet configuration
+
+parameter_types! {
+	// One storage item; key size 32, value size 8; .
+	pub const ProxyDepositBase: Balance = KILT;
```

Comment:
Question for @weichweich and @Diiaablo95: Do we want to merge with the current deposits and discuss later or have a discussion prior to approving? I think we should follow Kusama here which has a byte dependency

```rust
	// One storage item; key size 32, value size 8; .
	pub const ProxyDepositBase: Balance = deposit(1, 8);
	// Additional storage item size of 33 bytes.
	pub const ProxyDepositFactor: Balance = deposit(0, 33);
	pub const MaxProxies: u16 = 32;
	pub const AnnouncementDepositBase: Balance = deposit(1, 8);
	pub const AnnouncementDepositFactor: Balance = deposit(0, 66);
	pub const MaxPending: u16 = 32;
```
Where
```rust
	pub const fn deposit(items: u32, bytes: u32) -> Balance {
		items as Balance * 2_000 * CENTS + (bytes as Balance) * 100 * MILLICENTS
	}
```

Reply:
I think if we change this, it should be changed for all deposit values and not only for the proxy related ones. Therefore I'm against including this change in this PR, but would volunteer to create a follow up PR addressing this refactor of the runtimes.

Reply:
We should probably have a quick chat about this, update the PR, and then merge it. Maybe shortly after lunch?

Reply:
also agree. Make them high and safe in this PR (as they are right now). Depend on byte size&storage items in a separate PR.